cscs-checks/microbenchmarks/kernel_latency/kernel_latency.py

```diff
@@ -66,11 +72,8 @@ def __init__(self, kernel_version):
                     'latency': (6.6, None, 0.10, 'us')
                 },
                 'kesch:cn': {
-                    'latency': (12.0, None, 0.10, 'us')
+                    'latency': (13.7, None, 0.10, 'us')
```

Comment:
I wouldn't change that in this PR. It is unrelated to Tsa/Arolla porting and if it's an issue, it should be investigated separately.

Reply:
Ignore my comment.